en/docs/learn/exporting-user-info-to-zoho.md

```diff
@@ -0,0 +1,38 @@
+#Exporting WSO2 Identity Server User Information to Zoho CRM
+
+Zoho is a web-based online office suite containing word processing, spreadsheets, presentations, databases, note-taking, 
+wikis, web conferencing, customer relationship management, project management, invoicing, and other applications. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to Zoho. 
+
+Let's get started!
+
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+    
+##Try it Out
+
+1. Login to [Zoho](https://accounts.zoho.com/signin?servicename=ZohoHome&signupurl=https://www.zoho.com/signup.html).
+2. Click on **Contacts**.
+
+    ![contacts-on-zoho-tool-bar](../assets/img/tutorials/contacts-on-zoho-tool-bar.png)
+    
+3. Click **Import > Import Contacts**.
+
+    ![import-contacts-in-zoho](../assets/img/tutorials/import-contacts-in-zoho.png)
+    
+4. Click **Browse** in **From File** section.
+
+    ![browse-in-from-file-section](../assets/img/tutorials/browse-in-from-file-section.png)
+    
+5. Select the created **users.csv** file and click **Next**.
+6. Configure **What do you want to do with the records in the file?** section accordingly.
+7. In the next section map **Field Names** in the file with **Fields in Zoho CRM**. 
+
+    ![map-import-fields-to-zoho-crm](../assets/img/tutorials/map-import-fields-to-zoho-crm.png)
+    
+8. Click **Next**.
+9. Now you have successfully imported the user data into Zoho CRM. 
```

Comment:
We don't need to number this last one.

Reply:
Fixed https://github.com/wso2/docs-is/pull/1761/commits/9c3530b330cf9e035e07d98e8b77f97156b166e5